src/index.html

```diff
@@ -4,9 +4,411 @@
+
+            </p>
+
+            <p class="card__price">
+              <span class="card__price-label">
+                Price:
+              </span>
+              <span class="card__price-value">
+                $2,199
+              </span>
+            </p>
+
+            <a
+              href="#buy"
+              class="card__buy-button"
+              data-qa="hover"
+            >
+             Buy
+            </a>
+
+          </article>
+
+          <article class="card" data-qa="card">
+            <img
+              src="images/imac.jpeg"
+              alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)"
+              class="card__photo"
+            >
+
+            <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+            <p class="card__code">Product code: 195434</p>
+
+            <p class="card__rating">
+              <span class="stars stars--4">
+                <span class="stars__star"></span>
+                <span class="stars__star"></span>
+                <span class="stars__star"></span>
+                <span class="stars__star"></span>
+                <span class="stars__star"></span>
+              </span>
+
+              <span class="card__review">Reviews: 5</span>
+
+            </p>
+
+            <p class="card__price">
+              <span class="card__price-label">
+                Price:
+              </span>
+              <span class="card__price-value">
+                $2,199
+              </span>
+            </p>
+
+            <a
+              href="#buy"
+              class="card__buy-button"
+              data-qa="hover"
+            >
+             Buy
+            </a>
+          </article>
+
+          <article class="card" data-qa="card">
+            <img
+              src="images/imac.jpeg"
+              alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)"
+              class="card__photo"
+            >
+
+            <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+            <p class="card__code">Product code: 195434</p>
+
+            <p class="card__rating">
+              <span class="stars stars--4">
+                <span class="stars__star"></span>
+                <span class="stars__star"></span>
+                <span class="stars__star"></span>
+                <span class="stars__star"></span>
+                <span class="stars__star"></span>
+              </span>
+
+              <span class="card__review">Reviews: 5</span>
+
+            </p>
+
+            <p class="card__price">
+              <span class="card__price-label">
+                Price:
+              </span>
+              <span class="card__price-value">
+                $2,199
+              </span>
+            </p>
+
+            <a
+              href="#buy"
+            class="card__buy-button"
+            data-qa="hover"
+            >
+             Buy
+            </a>
```

Comment:
apply code styling recommendations from checklist
```suggestion
            <a
              href="#buy"
              class="card__buy-button"
              data-qa="hover"
            >
              Buy
            </a>
```